GhostGroup/grid-styled
# bug(flex): Fix responsiveStyle param

### How does this PR make you feel (in animated GIF format)?
![sad](https://media.giphy.com/media/O5pd2oPjjcGFW/giphy.gif)

#### Prerequisites
- Is there any documentation that needs to be updated? **N/A**

#### Please provide a summary of changes and any background context...
The props passed to `responsiveStyles` for `flexWrap` were incorrect. The third param is the value to insert if the param is a boolean. This PR adds the 'wrap' value as the boolean value.

Closes #17 

---

### Technical Review
* [X] Is there enough test coverage?
* Ask questions, have a conversation!


#### How should this be manually tested (any data/API considerations)?
Checkout branch, make sure `flex-wrap` is being applied correctly when using the prop `flexWrap`
